cli/crates/cli/src/cli_input.rs

```diff
@@ -129,18 +131,78 @@ pub enum SubCommand {
     Unlink,
 }
 
+// TODO see if there's a way to do this automatically (https://github.com/clap-rs/clap/discussions/4921)
+pub trait ArgumentNames {
```

Comment:
I don't love this solution since it requires manually updating it if we change / add argument names, but I opted out of a partial macro here for now to keep it simple, especially if we end up automating it somehow